frontend/micro-ui/web/micro-ui-internals/packages/modules/AttendenceMgmt/src/components/inbox/InboxFilter.js

```diff
@@ -0,0 +1,95 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import { useTranslation } from "react-i18next";
+import { CloseSvg, DateRangeNew, FilterIcon, FormComposer, RefreshIcon } from "@egovernments/digit-ui-react-components";
+
+
+const InboxFilter = ({type, onFilterChange, onClose}) => {
+  const { t } = useTranslation();
+  const [localSearchParams, setLocalSearchParams] = useState(null);
+  const [disabled, setDisabled] = useState(true)
+
+  useEffect(() => {
+    localSearchParams ? setDisabled(false) : setDisabled(true)
+  }, [localSearchParams])
+  
+  const configs = [{
+    head: "",
+    body: [
+      {
+        type: "custom",
+        isMandatory: false,
+        withoutLabel: true,
+        populators: {
+          name: "musterRolldateRange",
+          customProps: { t, optionKey: "i18nKey" },
+          component: (props, customProps) => (
+            <DateRangeNew 
+              {...customProps} 
+              values={localSearchParams?.range} 
+              label="Muster Roll Date Range" 
```

Comment:
Please use only localisation label/code not direct message

Reply:
Do we have localisation labels available for this screen as well? since this is new functionality I thought labels will also be added in api integration task

Reply:
Added localization labels